ftdc/cmd/parser.go

```diff
@@ -165,25 +194,20 @@ func (gpw *gnuplotWriter) CompileAndClose() string {
 
 func main() {
 	if len(os.Args) < 2 {
-		// We are a CLI, it's appropriate to write to stdout.
-		//
-
 		nolintPrintln("Expected an FTDC filename. E.g: go run parser.go <path-to>/viam-server.ftdc")
 		return
 	}
 
 	ftdcFile, err := os.Open(os.Args[1])
 	if err != nil {
-		// We are a CLI, it's appropriate to write to stdout.
-		//
-
 		nolintPrintln("Error opening file. File:", os.Args[1], "Err:", err)
-
 		nolintPrintln("Expected an FTDC filename. E.g: go run parser.go <path-to>/viam-server.ftdc")
 		return
 	}
 
-	data, err := ftdc.Parse(ftdcFile)
+	logger := logging.NewDebugLogger("parser")
+	logger.SetLevel(logging.WARN)
```

Comment:
Replaced with a simple `NewLogger` and removed the `SetLevel` -- using the INFO default.